### ISSUE TITLE
Audio skipping issue model Y

### ISSUE THREAD

**Issue by chamsters** [marcone/teslausb#887]
### Describe the problem

Hi all,

I have a Pi zero 2W with a 256GB U3 rated microsd that I'm connecting to the car through the glovebox USB slot.

Connecting the pi to my PC and running a usb speed test gives 30MB/s + sequential. 

I have no issues with Dashcam / sentry mode - both record fine.

Problem is with audio. I have 320kb MP3s, nothing too crazy but a decent collection. The car navigates the folders and plays the music but every so often (every couple of mins) it skips, like it's buffering. Can last a second, 2 seconds and resume. 

Any idea what the issue could be??

### Device

Raspberry Pi Zero 2 W

### OS Image

Prebuilt TeslaUSB image

### Car Model

Model Y

### USB connection

Glove box

**Comment by nilsmau** [comment]
i am experiencing the same thing ... basically music becomes unusable...
**Device**
Raspberry Pi Zero W

**Comment by nilsmau** [comment]
@marcone, any advice? might it be a broken / corrupted file system of sorts? Is the pi zero W to slow? 
@chamsters, were you able to fix it?

**Comment by deangoldhill** [comment]
Yes I have noticed this too. I also use a PI Zero 2 W. It was perfect in the beginning and has got worse over time. Dashcam video files always have no problems.
I zero'd the card and reinstalled TeslaUSB and became perfect again, but a few months later same problem.
I suspect it's to do with continuously writing to the card and the way the file system works.
I use a SanDisk 128GB Extreme microSDXC, perhaps a different card? any suggestions?

**Comment by THX723** [comment]
It might be worth checking your log and see if SSD TRIM is happening and successful.

On a non-related note, are any of you noticing the tracks end (cut off) several seconds earlier than it should be?

**Comment by tejesh83** [comment]
Anyone figure this out yet?  Having the same problem on my model 3 highland with skipping.  I haven't particularly noticed the end of the track cutting off, but will keep an eye for that.  I mostly just get random skipping in the middle of the track.  

I only saw the trim command run on the camera mount point, so tried mounting the music mount point and running trim on it to see if it makes any difference.  Will keep an eye on it for the next few rides.

**Comment by rvs007** [comment]
I'm also experiencing audio skipping with my setup using a Raspi4B.  I haven't made it to the end of a song to notice the end of track being cut off.  It's essentially not usable with the amount of skipping/pauses.  I hope someone can figure out the problem.

**Comment by satanforaday** [comment]
Same skipping issue with a Raspberry Pi 4 Model B Rev 1.5. Was going to recreate my SD card but if other people are having this issue, don't see a fresh install is going to fix anything.
